tests/Unit/Network/Observer/ConnectionObserverCollectionTest.php

```diff
@@ -6,7 +6,7 @@
  * file that was distributed with this source code.
  */
 
-namespace Stomp\Tests\Unit\Network\Observer\Heartbeat;
+namespace Stomp\Tests\Unit\Network\Observer;
```

Comment:
The corresponding system under test resides in [`Stomp\Network\Observer\ConnectionObserverCollection`](https://github.com/stomp-php/stomp-php/blob/ed6eb49b5351a8ddf46817312f69eb6a42c129ba/src/Network/Observer/ConnectionObserverCollection.php).